Modules/Devices/src/Devices.Domain/Entities/Identities/IdentityDeletionProcessAuditLogEntry.cs

```diff
@@ -184,11 +183,28 @@ public static IdentityDeletionProcessAuditLogEntry GracePeriodReminder3Sent(Iden
     {
         return new IdentityDeletionProcessAuditLogEntry(
             processId,
-            "The third grace period reminder notification has been sent.",
+            MessageKey.GracePeriodReminder3Sent,
             Hasher.HashUtf8(identityAddress.Value),
             null,
             DeletionProcessStatus.Approved,
             DeletionProcessStatus.Approved
         );
     }
 }
+
+public enum MessageKey
+{
+    ProcessStartedByOwner = 1,
+    ProcessStartedBySupport = 2,
+    ProcessApproved = 3,
+    ProcessRejected = 4,
+    ProcessCancelledByOwner = 5,
+    ProcessCancelledBySupport = 6,
+    ProcessCancelledAutomatically = 7,
+    ApprovalReminder1Sent = 8,
+    ApprovalReminder2Sent = 9,
+    ApprovalReminder3Sent = 10,
+    GracePeriodReminder1Sent = 11,
+    GracePeriodReminder2Sent = 12,
+    GracePeriodReminder3Sent = 13
```

Comment:
I don't think we need the `Process` prefix.

Remember to update the migrations accordingly.

```suggestion
    StartedByOwner = 1,
    StartedBySupport = 2,
    Approved = 3,
    Rejected = 4,
    CancelledByOwner = 5,
    CancelledBySupport = 6,
    CancelledAutomatically = 7,
    ApprovalReminder1Sent = 8,
    ApprovalReminder2Sent = 9,
    ApprovalReminder3Sent = 10,
    GracePeriodReminder1Sent = 11,
    GracePeriodReminder2Sent = 12,
    GracePeriodReminder3Sent = 13
```

Reply:
Changes made.